jupyterhub/templates/scheduler/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+{{ if .Values.scheduler.enabled -}}
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: scheduler
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ .Chart.Name }}-scheduler
+spec:
+  replicas: {{ .Values.scheduler.replicas }}
+  template:
+    metadata:
+      annotations:
+        # This lets us autorestart when the configmap changes!
+        checksum/config-map: {{ include (print $.Template.BasePath "/scheduler/configmap.yaml") . | sha256sum }}
+      labels:
+        chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+        component: scheduler
+        heritage: {{ .Release.Service }}
+        release: {{ .Release.Name }}
+    spec:
+      volumes:
+      - name: config
+        configMap:
+          name: {{ .Chart.Name }}-scheduler-config
+      containers:
+      - command:
+        - /usr/local/bin/kube-scheduler
+        - --address=0.0.0.0
+        - --leader-elect=true
+        - --scheduler-name={{ .Chart.Name }}-{{ .Release.Name }}-scheduler
+        - --lock-object-namespace={{ .Release.Namespace }}
+        - --lock-object-name={{ .Chart.Name }}-{{ .Release.Name }}-scheduler-lock
+        - -v=4
```

Comment:
I did not see information about this in the documentation, it some verbosity level i figure.